lib/prepare_config.js

```diff
@@ -1,12 +1,19 @@
 var registerLoader = require('./register_loader');
-var findCwd = require('./find_cwd');
+var fs = require('fs');
 
-function prepareConfig(self, env, opts) {
-  var basedir = findCwd(opts);
+function prepareConfig(self, env) {
+  var basedir = env.cwd;
   env.require.filter(toUnique).forEach(function(module) {
     self.requireLocal(module, basedir);
   });
 
+  if (env.configPath) {
+    // resolve symlink if needed
+    if (fs.lstatSync(env.configPath).isSymbolicLink()) {
```

Comment:
does this need to be done or would `realpathSync` always return the final path of the file? (I'm not sure the behavior)

Reply:
You're right. `realpathSync` returns the real path whether the given path is a symbolic link or not.

BTW, `require` seemed to resolve symbolic links for itself. 

```
$ touch a.js
$ ln -s a.js b.js
$ node
> require('./b.js')
{}
> require.cache
{ '/path/to/a.js':  { ... } }
> 
```

Now I doubt if this symbolic link  resolving is needed. Can we remove this?

Reply:
I didn't know that `require` resolved symlinks!  This can probably be removed then